tools/benchmark/cmd/util.go

```diff
@@ -118,12 +79,6 @@ func mustCreateConn() *clientv3.Client {
 	}
 
 	client, err := clientv3.New(cfg)
-	if targetLeader && len(leaderEps) == 0 {
```

Comment:
https://github.com/etcd-io/etcd/blob/ad9d69071936b5771830add74a799f2e822e2ffc/tools/benchmark/cmd/root.go#L77

@serathius @ahrtr do we still need this flag? I think that client side can pick up leader endpoint by themself.